docs/introduction.rst

```diff
@@ -29,11 +29,11 @@ receiving WebSocket frames) can't trigger this.
 How does ASGI work?
 -------------------
 
-ASGI is structured as a double-callable - the first callable takes a ``scope``,
-which contains details about the incoming request, and returns a second,
-coroutine callable. This second callable gets given ``send`` and ``receive``
-awaitables that allow the coroutine to monitor incoming events and send
-outgoing events.
+ASGI is structured as a double-callable---the first callable takes a
```

Comment:
Not a huge fan of en or em dashes, they have confusing subtlety for those unfamiliar with them. Let's keep the normal dashes, and use semicolons or periods if a bigger context break is needed?

Reply:
Sounds good to me; this specific line is what put me on that path since "double-callable - the first" was visually busy. I'll back most of these dash changes out and replace with sentence breaks where it makes sense!